### PR TITLE
Closes #18484: Reorganize settings Search, Tabs and Customize sub-pages margins

### LINKED ISSUE #18484: Reorganize settings sub-pages margins
To match the new look and feel, we need to reorganize margins on the settings sub-pages, related https://github.com/mozilla-mobile/fenix/issues/18132

### PR DESCRIPTION
**Pull request by Amejia481**
This PR updates the margins for Search, Tabs and Customize sub settings.

https://user-images.githubusercontent.com/773158/111364218-5756ec80-8667-11eb-84f1-aa768042d779.mp4

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by Amejia481** [comment]
@brampitoyo I updated the margins for the Search, Tabs and Customize sub settings pages, when you have some time please take a look :).

File: app/src/main/java/org/mozilla/fenix/settings/CustomizationFragment.kt [hunk 1]
```diff
@@ -33,7 +33,12 @@ class CustomizationFragment : PreferenceFragmentCompat() {
...
     override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
-        setPreferencesFromResource(R.xml.customization_preferences, rootKey)
+        val preferencesId = if (FeatureFlags.newIconSet) {
+            R.xml.customization_preferences_new_look
+        } else {
+            R.xml.customization_preferences
+        }
+        setPreferencesFromResource(preferencesId, rootKey)
 
         setupPreferences()
```
Comment (eliserichards) [inline app/src/main/java/org/mozilla/fenix/settings/CustomizationFragment.kt:43]: Do we need both `setPreferencesFromResource` and `setupPreferences`?

**Review by eliserichards — APPROVED** [review]
Looks good!

**Comment by Amejia481** [comment]
Thanks for the review @eliserichards but talking with @brampitoyo he confirmed me that we are not going to proceed with this change for the moment. I will proceed to close.

**Pull request closed without merging**